esp/src/eclwatch/DynamicESDLMethodWidget.js

```diff
@@ -50,12 +50,12 @@ define([
         init: function (params) {
             if (this.inherited(arguments))
                 return;
-
-            this.refresh();
+            this.refresh(params);
             this._refreshActionState();
         },
 
-        refresh: function () {
+        refresh: function (params) {
+            this._params = params;
             this.refreshGrid();
```

Comment:
You may want to move the call to ```this._refreshActionState();``` to inside the refresh function?

Reply:
Changed. Is this call required even if no event calls are being made on this particular grid?